Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -726,7 +539,7 @@
   MmServicesTableLib|MdePkg/Library/MmServicesTableLib/MmServicesTableLib.inf
   SmmServicesTableLib|MdePkg/Library/SmmServicesTableLib/SmmServicesTableLib.inf
   CpuExceptionHandlerLib|UefiCpuPkg/Library/CpuExceptionHandlerLib/SmmCpuExceptionHandlerLib.inf
-  BaseCryptLib|CryptoPkg/Library/BaseCryptLib/SmmCryptLib.inf
+  #BaseCryptLib|CryptoPkg/Library/BaseCryptLib/SmmCryptLib.inf

```

Comment:
Was this meant to be left commented? Please remove line if not needed.

Reply:
I accidentally left that in.  It's removed now.